plugins/incremental-ingestion-backend/src/database/migrations.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { resolvePackagePath } from '@backstage/backend-common';
+import { Knex } from 'knex';
+
+/** @public */
+export async function applyDatabaseMigrations(knex: Knex): Promise<void> {
+  const migrationsDir = resolvePackagePath(
+    '@devex/backend-incremental-ingestion',
```

Comment:
This should be `@backstage/plugin-incremental-ingestion-backend`

Reply:
Oops! Good catch. Fixing now...